### ISSUE TITLE
Modeling

### ISSUE THREAD

**Issue by sheacon** [sheacon/used_car_pricing#8]
Model Categories
---
### Parametric
- Linear/polynomial regression (for baseline)

### Tree-Based
- XGBoost, **CatBoost**, LightGBM
- Gradient boosting explained [1](https://learning.oreilly.com/library/view/hands-on-machine-learning/9781492032632/ch07.html#idm45022162248888) [2](https://bradleyboehmke.github.io/HOML/gbm.html)

### Deep Learning
- Transformer features --> TBD Model Head
- [SAINT](https://arxiv.org/abs/2106.01342) / [TabTransformer](https://arxiv.org/abs/2012.06678)
- [Meta-Analysis](https://m-clark.github.io/posts/2022-04-01-more-dl-for-tabular/)

**Comment by sheacon** [comment]
Features
- inventory
- recent change in inventory
- price change history
- days on market

**Comment by sheacon** [comment]
Uncertainty:
- model distributions or include measure of uncertainty
- interested in estimate of value but also risk-averse

**Comment by sheacon** [comment]
Ideally I'm going to have listings data and registrations data. Listings is helpful for market conditions and more individual vehicle details (options, condition, etc). Registrations provide final transaction

**Comment by sheacon** [comment]
Do I want to model cars that don't sell in some way? Some cars are unsellable...

**Comment by sheacon** [comment]
Impute options/features on car registrations without listings:
- mode of options for the same make/model/year/(VIN subset)
